python.pri

```diff
@@ -0,0 +1,35 @@
+win32 {
+	PYTHON=python
+} else {
+	PYTHON=$$system(which python 2>/dev/null)
```

Comment:
Do we really have to check anything but python, python2 and python3? Not to fond of listing every possible python 3 version there'll ever be in here.